packages/components/src/lozenge/docs/Lozenge.stories.mdx

```diff
@@ -12,18 +12,35 @@ import { Text } from "@components/typography";
 
 # Lozenge
 
-<Tagline>Lozenges are used to highlight an item's status for quick recognition or to give additional fast short information such as highlighting a new item.</Tagline>
+<Tagline>Lozenges are used to highlight an item's status, for quick recognition or to give additional information such as highlighting a new item.</Tagline>
 
 <ComponentInfo
     usage={"import { Lozenge } from \"@sharegate/orbit-ui\";"}
     githubPath="/packages/components/src/lozenge/src"
 />
 
+## Best Practices
+
+- Use to highlight an element in a list.
+- Use to bring attention to a particular element that requires the user’s attention, such as an expiring license or a problematic situation.
```

Comment:
The other scenario we use it for is to highlight something new, not sure it's relevant to add it as an example but if you think it is...